test/lib/pgBackRestTest/Module/Mock/MockAllTest.pm

```diff
@@ -81,13 +81,14 @@ sub run
         my $bS3 = $rhRun->{s3};
         my $bEncrypt = $rhRun->{encrypt};
         my $bDeltaBackup = $rhRun->{delta};
+        my $strCompressType = $rhRun->{compress};
 
         # Increment the run, log, and decide whether this unit test should be run
         if (!$self->begin("rmt ${bRemote}, s3 ${bS3}, enc ${bEncrypt}, delta ${bDeltaBackup}")) {next}
 
         # Create hosts, file object, and config
         my ($oHostDbMaster, $oHostDbStandby, $oHostBackup, $oHostS3) = $self->setup(
-            true, $self->expect(), {bHostBackup => $bRemote, bCompress => false, bS3 => $bS3, bRepoEncrypt => $bEncrypt});
+            true, $self->expect(), {bHostBackup => $bRemote, bS3 => $bS3, bRepoEncrypt => $bEncrypt});
```

Comment:
So this was bCompress = false before. In the MockArchiveStopTest.pm we set the $rhRun compress => undef to simulate that (I think) so why is it we're not setting it to  $rhRun compress => undef in these tests. I think we're going to have to discuss this so I can understand when to use undef and when not...

Reply:
`undef` = `compress=n`.  See above.

Reply:
Improved as above.